tests/Aspire.Dashboard.Tests/TelemetryRepositoryTests/TraceTests.cs

```diff
@@ -196,17 +271,15 @@ public void AddTraces_Traces_MultipleOutOrOrder()
             {
                 AssertId("1", trace.TraceId);
                 AssertId("1-1", trace.FirstSpan.SpanId);
-                AssertId("", trace.FirstSpan.ScopeName);
+                Assert.Equal("", trace.FirstSpan.Scope.ScopeName);
```

Comment:
Just curious - why the change from `AssertId` to `Assert.Equal`?

Reply:
`AssertId` does some work to convert hex encoded bytes to a UTF8 string. It was wrongly being used here. It only worked before the scope name happened to be an empty string.